Dockerfile

```diff
@@ -56,7 +56,14 @@ RUN apt update && apt install -y \
     openssl \
     && rm -rf /var/lib/apt/lists/*
 
+# Add tini, which is an init process that handles signaling within the container
```

Comment:
What does [that](https://github.com/krallin/tini?tab=readme-ov-file#using-tini) mean?

> NOTE: If you are using Docker 1.13 or greater, Tini is included in Docker itself. This includes all versions of Docker CE. To enable Tini, just pass the --init flag to docker run.

Do we actually need to add _Tini_?

Reply:
From my understanding, it is included in the docker Daemon, not every docker images itself. I think adding it to the image is better, so it's transparent for the end user. It would be possible to just add `init: true` to the docker compose files, and add to the documentation to add `--init true` to the `docker run` command. IMO it's not a huge difference as Tini is, well, tiny, so the docker image size is barely increased.